src/components/RecurringEvent/index.js

```diff
@@ -151,39 +205,41 @@ class RecurringEvent extends React.Component {
                         <span className="glyphicon glyphicon-remove"></span>
                     </button>
                     <h2>Toistuva tapahtuma</h2>
```

Comment:
Should we have just HelTimePicker (no DateTimeField) with validation here, so the user can see and set the time of the repeating events they are creating? Or does the user realize they have to set the time for the repeat before opening the modal?